analyses/TX_cd_2010/03_sim_TX_cd_2010.R

```diff
@@ -0,0 +1,350 @@
+###############################################################################
+# Simulate plans for `TX_cd_2010`
+# © ALARM Project, December 2022
+###############################################################################
+
+cluster_pop_tol <- 0.0025
+nsims <- 12500
+pop_temp <- 0.03
+sa_city <- 0.99
+sa <- 0.95
+
+# Unique ID for each row, will use later to reconnect pieces
+map$row_id <- 1:nrow(map)
+
+# Run the simulation -----
+cli_process_start("Running simulations for {.pkg TX_cd_2010}")
+
+########################################################################
+# Cluster #1: Greater Houston
+
+# https://www.dshs.texas.gov/center-health-statistics/center-health-statistics-texas-county-numbers-public-health-regions#:~:text=The%20county%20FIPS%20(Federal%20Information,county%20FIPS%20code%20of%2048xxx.
+# county code * 2 - 1
+clust1 <- c("015", "039", "071", "157",
+    "167", "201", "291", "339", "473")
+
+m1 <- map %>% filter(county %in% clust1)
+attr(m1, "pop_bounds") <-  attr(map, "pop_bounds")
+m1 <- set_pop_tol(m1, cluster_pop_tol)
```

Comment:
I might be missing something, but I don't think you need one of line 27 or 28. 28 overwrites 27, *I think*. It's possible something is fishy with the mean that makes this necessary.

Reply:
I think this is an artifact of 2020 where I did the same thing. I don't think it's necessary, and comes from me doing this manually before I knew `set_pop_tol()` existed.